### PR TITLE
Community page hi

### PR DESCRIPTION
**Pull request by harish343**
Fixes :  #39596
New feature has been added communtiy page in hindi please review it and ping me if any changes had to be made on this repository.

### CODE REVIEW

File: content/hi/community/_index.html [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+layout: basic
+cid: community
+community_styles_migrated: true
+---
+<img
+  id="banner"
+  srcset="/images/community/kubernetes-community-final-02.jpg 1500w, /images/community/kubernetes-community-02-mobile.jpg 900w"
+  sizes="(max-width: 900px) 900px, (max-width: 1920px) 1500px"
+  src="/images/community/kubernetes-community-final-02.jpg"
+  alt="Kubernetes conference photo">
```
Comment (sftim) [inline content/hi/community/_index.html:12]: nit: you could localize this alt attribute

File: content/hi/community/_index.html [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+  <div class="community-nav-item">
+    <a href="/releases">रिलीज़ेज़</a>
+  </div>
+</div>
+
+<div class="community-section" id="gallery">
+  <img src="/images/community/kubernetes-community-final-03.jpg" alt="Kubernetes conference gallery photo" class="community-gallery-desktop">
+  <img src="/images/community/kubernetes-community-final-04.jpg" alt="Kubernetes conference gallery photo" class="community-gallery-desktop">
+  <img src="/images/community/kubernetes-community-final-05.jpg" alt="Kubernetes conference gallery photo" class="community-gallery-desktop">
+  <img src="/images/community/kubernetes-community-04-mobile.jpg" alt="Kubernetes conference gallery photo" class="community-gallery-mobile">
```
Comment (sftim) [inline content/hi/community/_index.html:51]: nit: you could localize these alt attributes

File: content/hi/community/_index.html [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+
+<div id="resources" class="community-section">
+  <h2>चर्चाएं</h2>
+
+  <p class="community-simple">हम बहुत बातें करते हैं। इन सभी प्लेटफ़ॉर्मों पर खोजें और बातचीत में शामिल हों।</p>
+
+  <div class="container">
+    <div class="community-resource">
+      <a href="https://discuss.kubernetes.io/">
+        <img src="/images/community/discuss.png" alt="Forum">
```
Comment (sftim) [inline content/hi/community/_index.html:109]: nit: you could localize this alt attribute

File: content/hi/community/_index.html [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+      <details><summary><em>निमंत्रण चाहिए?</em></summary>
+        आमंत्रण के लिए  <a href="https://slack.k8s.io/">https://slack.k8s.io/</a>
+        पर जाएं।</details>
+    </div>
+  </div>
+</div>
+
+<div class="community-section" id="events">
+  <div class="container">
+   <h2>Upcoming Events</h2>
```
Comment (sftim) [inline content/hi/community/_index.html:155]: nit: you could localize this title

**Review by sftim — COMMENTED** [review]
A few extra things to localize if you'd like to

**Comment by jihoon-seo** [comment]
It seems that you opened another PR #39672 with reflecting review comments within it, so let me mark this PR as a duplicate.
/triage duplicate

**Comment by Babapool** [comment]
Since PR #39672 is active and under review (and this is marked as duplicate PR), hence closing this PR.

/close

**Pull request closed without merging**